reserved_keyword_checker/tests/test_app/non_concrete_app/models.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+class BasicModel(models.Model):
+    class Meta:
+        app_label = 'non_concrete_app'
+
+    first_name = models.CharField(max_length=20)
+
+
+class ModelMixin(object):
+    age = 100
+
+
+class MixedModel(BasicModel, ModelMixin):
+    class Meta:
+        app_label = 'non_concrete_app'
+
+    last_name = models.CharField(max_length=20)
+
+
+class AbstractModel(models.Model):
+    class Meta:
+        app_label = 'non_concrete_app'
+
+    start_date = models.CharField(max_length=20)
+
+    class Meta:
+        abstract = True
```

Comment:
This contained `Meta` class should be combined with the other one above?

Reply:
Not sure how that slipped by. Thanks